### ISSUE TITLE
Problems with timeout and inability to configure from HASS

### ISSUE THREAD

**Issue by robigan** [h4de5/home-assistant-vimar#65]
I seem to have some network problems with Vimar with logs showing up that timeouts occur and devices being temporarily unavailable in Home Assistant. I am aware that before the new UI config feature, I was able to change timeout, then the move to the new UI config feature occurred and timeouts weren't a problem due to the switch having implemented my timeout config. Despite all this, recently my hass install got nuked, so I had to restart from scratch, **but I can't find any options for timeout?**

**Comment by h4de5** [comment]
if you mean that the vimar webserver is temporarily not available as well it could be related to #62

**Comment by robigan** [comment]
> if you mean that the vimar webserver is temporarily not available as well it could be related to #62

I don't believe it crashes the server as it's able to pull new data within a second or two once I see in the UI that it has gone unavailable

**Comment by h4de5** [comment]
once you setup the vimar platform, you can configure it with the UI. 

<img width="418" alt="image" src="https://github.com/h4de5/home-assistant-vimar/assets/6115324/a7e23b6d-f52b-46a4-8488-ce06ff772e98">

**Comment by robigan** [comment]
> once you setup the vimar platform, you can configure it with the UI. 
> 
> 
> 
> <img width="418" alt="image" src="https://github.com/h4de5/home-assistant-vimar/assets/6115324/a7e23b6d-f52b-46a4-8488-ce06ff772e98">
> 
> 

Yeah I can’t find this UI screen that’s my issue

**Comment by robigan** [comment]
> > if you mean that the vimar webserver is temporarily not available as well it could be related to #62
> 
> I don't believe it crashes the server as it's able to pull new data within a second or two once I see in the UI that it has gone unavailable. But if the webserver is capable of restarting in such little time, then yes it might be crashing as the host goes unreachable. Despite this, the service is available 80% of the time
> 
> ```
> Error occurred: HTTPSConnectionPool(host='192.168.1.110', port=443): Max retries exceeded with url: /cgi-bin/dpadws (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f96e8c10b50>: Failed to establish a new connection: [Errno 113] Host is unreachable'))
> ```

I have just ran the logs through ChatGPT's code interpreter and these are the results for "System has Started"
```
2022-04-09    1
2022-09-11    2
2022-10-30    1
2022-11-17    3
2022-12-04    2
2022-12-27    1
2023-01-01    5
2023-01-02    1
2023-04-10    1
2023-04-22    1
2023-06-07    1
Name: Data, dtype: int64
```

So I doubt it's crashing

**Comment by h4de5** [comment]
do you have a Configure link within the vimar integration page: https://your.host:port/config/integrations/integration/vimar
![image](https://github.com/h4de5/home-assistant-vimar/assets/6115324/4aeaa5c6-0949-4578-a397-5f322c939d76)

it should show the regular connection details first, and after that all further options like above.

if this is not working - you can try to manually change the config options in:

> config/core.config_entries

search for: 
```
"domain": "vimar",
```

right below that, there should be an setting like this:

```
"timeout": 6,
```

you can change this, and restart HA.
KEEP IN MIND: please make a copy of your config directory before you directly change anything there. breaking this will damage your HA installation.

**Comment by robigan** [comment]
Yeah I am aware of manual configuration like that but I've seen frowning upon doing it like that in the forums. And thanks for showing how to change timeout but the same host error occurs, I am gonna set my Hass install to use Host network resources as it's hosted in a privatized docker container
